### ISSUE TITLE
[Accessibility][PDF Viewer] The Adobe Acrobat link in the controls dock has no name or role

### ISSUE THREAD

**Issue by LevelAccessA11y** [adobe/aem-core-wcm-components#1738]
**Steps to Reproduce**
1. Enable NVDA.
2. Open the page in Google Chrome.
3. Navigate through elements in the control dock using arrow keys.

**Current Behavior**
The Adobe Acrobat icon controls can be clicked on with a mouse to open the Adobe Acrobat Home page in a new tab, but are marked up as <div> elements and do not provide name or interactive role information. No textual alternative is provided for the Adobe Acrobat icon. Screen reader users and keyboard-only users will be unable to use these links, and users of voice input will be unable to target the links directly.

**Expected behavior/code**
All interactive elements provide name and role information.

**Environment**
Chrome 91, Windows 10, NVDA 2020.4 
AEM Core Components v. 2.17  

**Affected Users**
Assistive technology users, keyboard users

**WCAG SC**
4.1.2 Name, Role, Value

**Code snippet**
```
<div class="sdk-Branding-branding sdk-Branding-hudBranding" role="presentation"><svg data-name="Layer 1" viewBox="0 0 24 24" focusable="false" aria-hidden="true" role="img" class="spectrum-Icon spectrum-Icon--sizeM">...</svg></div>
```

**Possible Solution**
If possible, use a native HTML anchor element with a valid href value, which will provide role information (and keyboard accessibility) by default. 

If this is not possible, provide appropriate role information (in this case, role="link"). Ensure a textual alternative is provided for the Adobe Acrobat icon so the link has appropriate textual content.

**Comment by richardhand** [comment]
@adobe export issue to Jira project SITES

**Comment by glasert** [comment]
This issue won't be fixed as the acrobat team is working on a modern viewer which addresses accessibility and will replace the current viewer in 2022. Therefore we defer and close this issue
